src/Holidays/Orphans.hs

```diff
@@ -0,0 +1,15 @@
+{-# OPTIONS -Wno-orphans #-}
+
+module Holidays.Orphans () where
+
+import Data.ByteString.Char8
+import Data.Csv
+import Data.Time
+import Language.Haskell.TH.Lift
+
+deriveLift ''Day
+deriveLift ''Only
+
+instance FromField Day where
```

Comment:
Orphans in a library? It might have unexpected consequences.
Consider exposing something like `readHolidaysFromFile :: FilePath -> IO (Set Day)`. Or make a newtype over `Day` (and coerce after reading to get rid of it)

Reply:
Yeah, true